docs/en/architecture.md

```diff
@@ -1,71 +1,55 @@
+This solution deploys the Amazon CloudFormation template in your AWS Cloud account and provides three parts:
 
 - Notebook Experiment
 - Batch Evaluation
 - Visualization
 
 **Notebook Experiment**
 
-1.  The solution deploys an instance for 
-[AWS SageMaker Notebook](https://docs.aws.amazon.com/sagemaker/latest/dg/nbi.html). 
-The user can do **Notebook Experiment** for drug discovery on classical computing and 
-quantum computing(1).
+1. The solution deploys an instance for [AWS SageMaker Notebook](https://docs.aws.amazon.com/sagemaker/latest/dg/nbi.html), which allows **Notebook Experiment** for drug discovery(1).
 
-2. The notebook comes with prepared sample code for different problems 
-in drug discovery, like molecular unfolding, molecule simulation and so on. 
-The user can learn how to study these problems based on classical 
-or quantum computing resource through 
-[Amazon Braket][braket]. The step-by-step guide is 
-provided in the [Workshop Page](workshop/background.md)(2).
+2. The notebook comes with prepared sample code for different problems in drug discovery, such as molecular unfolding, molecule simulation and so on. You can learn how to study these problems based on classical computing or quantum computing through [Amazon Braket][braket]. For details, refer to [Workshop](workshop/background.md)(2).
 
-3. The solution creates [NAT gateways][nat] in public [subnets][subnet], which connects to internet using an [internet gateway][internet-gateway]. The Notebook is deployed in private subnets, which can access internet using NAT gateways(3).
+3. The solution creates [NAT gateways][nat] in public [subnets][subnet], which connects to internet using an [internet gateway][internet-gateway]. The notebook instance is deployed in private subnets, which can access internet using NAT gateways(3).
 
 **Batch Evaluation**
 
-1. The solution deploys [AWS Step Functions][step-functions] workflows for users to do **Batch Evaluation**(4). 
+1. The solution deploys [AWS Step Functions][step-functions] workflows for **Batch Evaluation**(4). 
 
-2. The AWS Step Functions workflows launches various computing tasks in parallel through 
-    [AWS Batch][batch] jobs based on different model parameters, resources, classical computing or quantum computing(5).
+2. The AWS Step Functions workflow launch various computing tasks in parallel through [AWS Batch][batch] jobs based on different model parameters, resources, classical computing or quantum computing(5).
 
-3. Each AWS Batch job which uses the pre-built container image in [Amazon ECR][ecr](7) tries to evaluate a particular drug discovery problem based on a specific model parameter(6). 
+3. Each AWS Batch job uses the pre-built container image in [Amazon ECR][ecr](7) and attempts to evaluate a particular drug discovery problem based on specific model parameters(6). 
 
 4. For classical computing, AWS Batch jobs evaluate the problem locally, and save results in [Amazon S3][s3](10).
```

Comment:
I think `locally` is inaccurate. should be `elastic compute cloud(Amazon EC2)` or else